CallWithDoxDialer.podspec

```diff
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   Doximity lets healthcare professionals make phone calls 
                   to patients while on the go -- without revealing personal phone numbers.
-                  Calls are routed through Doximity's HIPPA-secure platform and relayed to the patient
+                  Calls are routed through Doximity's HIPAA-secure platform and relayed to the patient
                   who will see the doctor's office number in the Caller ID.
                   Doximity Dialer is currently available to verified physicians, nurse practitioners,
```

Comment:
The Podspec has a `Doximity Dialer is currently available` reference that should be changed to just `Doximity is currently available` as well...